pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[project]
+name = "async-workflow-runner"
+version = "0.0.0"
+authors = [
+  { name="Erik Trygg", email="etrg@novonordisk.com" },
+]
+description = "Package for running and tracking workflows asynchronously, with various helper features to make it easier"
+readme = "README.md"
+requires-python = ">=3.9"
```

Comment:
Consider a newer version, as there's very few things in here yet?

Reply:
I know what you mean - cool to make use of the latest python features. For now I insist sticking with a minimum version 3.9, at least for all "core" packages that are intended to be incorporated in applications.

For any stand-alone applications, we have no reason to start looking at newer versions.